src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -101,6 +101,9 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                                     <fa-icon [icon]="faChartBar"></fa-icon>&nbsp;<span jhiTranslate="statistics.statistics-title">Statistics</span>
                                 </a>
                             }
+                            <a [routerLink]="['/courses', programmingExercise.course!.id!, 'exercises', programmingExercise.id!]" class="btn btn-info btn-sm">
```

Comment:
Please do not use the non-null assertion operator (the exclamation mark). Because by doing so you're telling the compiler that this expression can't be null or undefined and the compiler should stop complaining about it! But in case it is undefined for whatever reason, this would completely crash the view.